code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2965,26 +2965,29 @@
 	if(..())
 		return 1
 
+	var/mob/living/carbon/human/H
```

Comment:
There's no need for these three lines - a human check is the first part of `isslimeperson`.